wherehows-web/app/components/dataset-author.js

```diff
@@ -174,6 +174,20 @@ export default Component.extend({
         updatedOwner,
         ...updatingOwners.slice(ownerPosition + 1)
       ];
+      // The list of ldap userNames currently in the list
+      const userNames = updatedOwners.mapBy('userName');
+      // Checks that the userNames are not already in the list of current owners
+      const hasDuplicates = new Set(userNames).size !== userNames.length;
+
+      if (hasDuplicates) {
+        set(
+          this,
+          'errorMessage',
+          `Uh oh! Looks like there are duplicates in the list of owners, please remove them first.`
```

Comment:
good catch thanks! @jgandu was going to put the offending `userName` in the list, but didn't think it was worth it